lib/models/application.js

```diff
@@ -492,6 +500,7 @@ const getApplicationModel = function (deps, opts) {
 		 * @param {String} [options.applicationType] - application type slug e.g. microservices-starter
 		 * @param {String} options.deviceType - device type slug
 		 * @param {(Number|String)} [options.parent] - parent application name or id
+		 * @param {(String|Number|null)} [options.organization] - handle (string) or id (number) of the organization that the application will belong to or null
```

Comment:
`organization` can't be `null`.

Reply:
I actually just checked today the API source for this
See: https://github.com/balena-io/balena-api/blob/b84ec51f8c20896c037297ab7d623a7bfb9c998b/src/models/hooks/resources/application.ts#L84-L102

but it's best to anyway start stricter :+1:

Reply:
But we've gone over this several times in calls and whatnot and agreed it had to be an argument that is explicitly provided. Do I misremember? That hook will also have to change but that's unrelated.

Reply:
No no, you are right, nice catch.